rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Constants.java

```diff
@@ -6,6 +6,7 @@
 public class Constants {
 	public static final String DYNAMO_TABLE_NAME_ENV_VARIABLE = "DYNAMO_TABLE_NAME";
 	public static final String KMS_KEY_ENV_VARIABLE = "KMS_KEY";
+	public static final String TEST_KMS_KEY_ENV_VARIABLE = "TEST_KMS_KEY";
```

Comment:
I don't think we should put a test based env var in the non-test `Constants` class.  Perhaps make a `TestConstants` class in `test-commons`.

Reply:
A class that's sole purpose to hold one constant doesn't sound too appealing to me. Maybe we just keep as a constant at the top of `CpcApiAcceptanceTest.java`?

Reply:
Good point.  I think moving it to the top of `CpcApiAcceptance.java` is best.